From: adminuser <adminuser@cvt.bln.roederer.dhs.org>
Date: Thu, 10 Dec 2020 23:00:46 +0100
Subject: ath79: deactivate 4k NOR-sectirs for tiny-target

even with some patches these a many jffs2 errors which break the fs.

Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.822287] jffs2: jffs2_scan_eraseblock(): Magic bitmask 0x1985 not found at 0x00035020: 0x5fce instead
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.831836] jffs2: jffs2_scan_eraseblock(): Magic bitmask 0x1985 not found at 0x00035038: 0xdb5f instead
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.846762] jffs2: jffs2_scan_inode_node(): CRC failed on node at 0x00035fe4: Read 0x51ec5501, calculated 0xe0a62d2
0
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.867808] jffs2: Empty flash at 0x000364fc ends at 0x00037000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.873792] jffs2: CLEANMARKER node found at 0x00037000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.887556] jffs2: Empty flash at 0x0003700c ends at 0x00038000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.893542] jffs2: CLEANMARKER node found at 0x00038000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.912456] jffs2: Empty flash at 0x0003800c ends at 0x00039000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.918438] jffs2: CLEANMARKER node found at 0x00039000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.932201] jffs2: Empty flash at 0x0003900c ends at 0x0003a000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.938179] jffs2: CLEANMARKER node found at 0x0003a000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.957094] jffs2: Empty flash at 0x0003a00c ends at 0x0003b000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.963070] jffs2: CLEANMARKER node found at 0x0003b000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   15.976831] jffs2: Empty flash at 0x0003b00c ends at 0x0003c000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   15.982817] jffs2: CLEANMARKER node found at 0x0003c000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   16.001739] jffs2: Empty flash at 0x0003c00c ends at 0x0003d000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   16.007721] jffs2: CLEANMARKER node found at 0x0003d000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   16.021499] jffs2: Empty flash at 0x0003d00c ends at 0x0003e000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   16.027479] jffs2: CLEANMARKER node found at 0x0003e000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.warn kernel: [   16.046396] jffs2: Empty flash at 0x0003e00c ends at 0x0003f000
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   16.052376] jffs2: CLEANMARKER node found at 0x0003f000, not first node in block (0x00030000)
Thu Dec 10 21:37:23 2020 kern.err kernel: [   16.089470] jffs2: Dirent at 00073fd0 has zeroes in name. Truncating to 12 chars
Thu Dec 10 21:37:23 2020 kern.notice kernel: [   16.096953] jffs2: jffs2_scan_dirent_node(): Name CRC failed on node at 0x00073fd0: Read 0x3e103acc, calculated 0x8
dc05bf1

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index b8e94ebe4edf2d933f91ba0d521786284aeadda7..01d93969e5328fbddf50730ac7340fcaf8d3771a 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -1,5 +1,4 @@
 CONFIG_LEDS_RESET=y
-CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
 CONFIG_NET_DEVLINK=y
 CONFIG_NET_DSA=y
 CONFIG_NET_DSA_MV88E6060=y
